docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -52,7 +51,7 @@ a model:
 
 For distributed multi-machine training, Determined will automatically detect a common network
 interface that is shared by the agent machines. If your cluster has multiple common network
-interfaces, we advise specifying the fastest one in :ref:`cluster-configuration` under
+interfaces, we advise specifying the fastest one in :ref:`internet-access` under
```

Comment:
Again, to this link is not a reference for the problem described in this sentence.

Reply:
Maybe link to `https://docs.determined.ai/latest/reference/deploy/config/master-config-reference.html#dtrain-network-interface` instead?

Reply:
edited.

For distributed multi-machine training, Determined will automatically detect a common network
interface that is shared by the agent machines. If your cluster has multiple common network
interfaces, we advise specifying the fastest one. To do this, you can edit the :ref:`network
interface <master-config-reference-dtrain-network-interface>`
``task_container_defaults.dtrain_network_interface`` option.